roachpb/batch_test.go

```diff
@@ -0,0 +1,41 @@
+// Copyright 2014 The Cockroach Authors.
```

Comment:
there are some other batch tests in `roachpb/api_test.go`. Can you just put your test there (for now)?